package.json

```diff
@@ -48,15 +48,17 @@
   "private": true,
   "dependencies": {},
   "devDependencies": {
+    "@types/jest": "^24.0.12",
```

Comment:
Should this be in here, or in the kolibri-tools package.json?

Reply:
Yes, it needs to be in the project `package.json`. It's only metadata exposing what `jest` provides. Having `jest` encapsulated within kolibri-tools is the reason I've added this.

Reply:
OK - is it an IDE plugin that is using this metadata? (as we are not using typescript anywhere in the project)

Reply:
I think most code-hinting uses TypeScript, even VSCode. https://github.com/felixfbecker/vscode-php-intellisense

Reply:
Sublime has not complained hitherto at the lack, so I hadn't noticed!